ubuntu_from_source/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Ubuntu image tags as they appear here: https://hub.docker.com/_/ubuntu. Example: UBUNTU_VERSION=20.04
+ARG UBUNTU_VERSION=20.04
+# ROOT version in the MAJOR.MINOR.REVISION format. Example: ROOT_VERSION=6.26.00
+ARG ROOT_VERSION
+# C++ standard to compile ROOT. Example: CMAKE_CXX_STANDARD=17
+ARG CMAKE_CXX_STANDARD
+
+FROM ubuntu:$UBUNTU_VERSION
+
+LABEL maintainer.name="ROOT team"
+LABEL maintainer.email="root-dev@cern.ch"
+
+ENV LANG=C.UTF-8
+
+COPY packages packages
+
+RUN apt-get update -qq && \
+    ln -sf /usr/share/zoneinfo/UTC /etc/localtime && \
+    apt-get -y install $(cat packages) wget git && \
+    apt-get autoremove -y && \
+    apt-get clean -y && \
+    rm -rf /var/cache/apt/archives/* && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV ROOTSYS /opt/root
+ENV PATH $ROOTSYS/bin:$PATH
+ENV PYTHONPATH $ROOTSYS/lib:$PYTHONPATH
+ENV CLING_STANDARD_PCH none
+
+RUN echo $ROOTSYS/lib >> /etc/ld.so.conf && \
+    ldconfig
+
+RUN yes | unminimize
+
+# clone build and install
+ARG ROOT_VERSION=6.26.00
+ARG CMAKE_CXX_STANDARD=17
+
+RUN ROOT_GIT_REVISION="v$(echo $ROOT_VERSION | cut -d. -f1)-$(echo $ROOT_VERSION | cut -d. -f2)-$(echo $ROOT_VERSION | cut -d. -f3)" \

```

Comment:
this way `master` or `latest-stable` can't be specified anymore.

maybe it would make more sense if the `ARG` was the tag or branch name that should be built, and then (if needed and if possible) we can extract the ROOT version from that?